### ISSUE TITLE
Net Core X86 X64 doesn't work

### ISSUE THREAD

**Issue by furkan69** [RumbleMike/ValorantStreamOverlay#67]
Hello, when i launch the overlay it tell me to install any dotnet but when i install the x86 it still ask the install the thing when i launch 

and with the x64 when i try to launch the overlay nothing happens nothing start. HEELPPP !!! 👍

**Comment by IFDES** [comment]
same

**Comment by reverie-ss** [comment]
![Untitled](https://user-images.githubusercontent.com/14177137/103474159-0bfea300-4dc7-11eb-8ccf-bcf1ca3e4848.png)

**Comment by Honetar19** [comment]
facing same problem, help please

**Comment by furkan69** [comment]
i uninstalled the x64 and installed this one instead https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-5.0.101-windows-x64-installer 

and the x86 still installed

**Comment by Whanos** [comment]
Exactly what version did you install @furkan69 ? @Honetar19 ? Please send me the links

**Comment by carlosmr14** [comment]
> Exactly what version did you install @furkan69 ? @Honetar19 ? Please send me the links

Same problem here. I use windows 10 Pro x64
I download these files :

https://dotnet.microsoft.com/download/dotnet/thank-you/sdk-5.0.101-windows-x64-installer
https://dotnet.microsoft.com/download/dotnet-core/3.1/runtime/?utm_source=getdotnetcore&utm_medium=referral

After install these files the program doesnt run, same message error

**Comment by forszaken** [comment]
@RumbleMike 
this problem is written here, maybe it will help you::
https://stackoverflow.com/questions/59367037/published-net-core-app-warns-to-install-net-core-but-its-already-installed

@carlosmr14 
This helped me, thanks a lot. It is worth adding to the documentation until the problem is resolved.

**Comment by Whanos** [comment]
Yes. Please install the .net core 3.1 runtime instead, that one is correct.
